lib/rubocop/cop/rails/safe_navigation.rb

```diff
@@ -42,30 +44,38 @@ class SafeNavigation < Cop
         MSG = 'Use safe navigation (`&.`) instead of `%s`.'.freeze
 
         def_node_matcher :try_call, <<-PATTERN
-          (send _ ${:try :try!} ...)
+          (send _ ${:try :try!} $_ ...)
         PATTERN
 
         def on_send(node)
           return if target_ruby_version < 2.3
 
-          try_call(node) do |method|
-            return if method == :try && !cop_config['ConvertTry']
-            add_offense(node, :expression, format(MSG, method))
+          try_call(node) do |try_method, method_to_try|
+            return if try_method == :try && !cop_config['ConvertTry']
+            return unless method_to_try.sym_type?
+            method, = *method_to_try
+            return unless method =~ /\w+[=!?]?/
```

Comment:
Does anyone know if there is regex similar to this that exists in another cop? It is matching against standard methods that are comprised of words, underscores, and end with `?`, `!`, or `=`.